src/simplify.rs

```diff
@@ -225,7 +231,8 @@ fn is_empty_line(buf: &str) -> bool {
     //
     // See discussion at: https://github.com/deltachat/deltachat-core-rust/pull/402#discussion_r317062392
```

Comment:
Maybe try the suggestion `buf.chars().all(|c| c.is_whitespace())` and remove this comment since simple check started to cause problems?